packages/venia-concept/src/components/FilterModal/FilterSearch/filterSearch.js

```diff
@@ -0,0 +1,88 @@
+import React, { Fragment } from 'react';
+import TextInput from 'src/components/TextInput';
+import Trigger from 'src/components/Trigger';
+import { Form } from 'informed';
+import Icon from 'src/components/Icon';
+import ClearIcon from 'react-feather/dist/icons/x';
+import SearchIcon from 'react-feather/dist/icons/search';
+import classify from 'src/classify';
+import defaultClasses from './filterSearch.css';
+
+const clearIcon = <Icon src={ClearIcon} size={18} />;
+const searchIcon = <Icon src={SearchIcon} size={18} />;
+
+const withFilterSearch = WrappedComponent => {
+    class withFilterSearch extends React.Component {
+        state = {
+            filterQuery: ''
+        };
+
+        handleFilterSearch = event => {
+            const { value } = event.currentTarget || event.srcElement;
+            this.setState({ filterQuery: value });
+        };
+
+        handleResetSearch = () => this.setState({ filterQuery: '' });
+
+        getFilteredItems = (items, filterQuery) =>
+            items.filter(
+                item =>
+                    item.label
+                        .toUpperCase()
+                        .indexOf(filterQuery.toUpperCase()) >= 0
```

Comment:
Consider using `.includes()`.

Reply:
`includes()` is case sensitive, this approach is case insensitive. Let me know if I should go with `includes`anyway.

Reply:
Thanks for the clarification. I have added `includes` instead of `indexOf` in the latest update.